### DISCUSSION TITLE
ERROR: ffmpeg not found. Please install or provide the path using --ffmpeg-location

### DISCUSSION THREAD

**Discussion by whoismeandwhoisyou** [jely2002/youtube-dl-gui#538] [General]
FFmpeg is definitely installed on my Mac; I verified it with the terminal command `ffmpeg -version` and confirmed the path with the command `which ffmpeg`. However, I can't find where to enter the FFmpeg path in the app to fix the error. Does anyone know how to resolve this?

**Comment by jely2002** [comment]
This has been fixed in v3, where it automatically retrieves the proper version!
